examples/gallery/histograms/blockm.py

```diff
@@ -0,0 +1,63 @@
+"""
+Blockmean
+---------
+The :meth:`pygmt.blockmean` method calculates different quantities
+inside blocks/bins whose dimensions are defined via the ``spacing`` parameter.
+The following examples show how to calculate averages of given values inside
+each block as well as how to report the number of points inside each bin.
```

Comment:
```suggestion
each block and how to report the number of points inside each bin.
```
Personal preference, but I think this reads better.